spec/fixtures/education_benefits_claims/0994/kitchen_sink.json

```diff
@@ -64,5 +64,6 @@
   "highestLevelofEducation": "other",
   "otherEducation": "Other Education Type",
   "privacyAgreementAccepted": true,
-  "expectedActiveDutyStatusChange": true
+  "expectedActiveDutyStatusChange": true,
+  "expectedReleaseDate": "2025-01-01"
 }
```

Comment:
Not a big deal but you might go ahead and throw in a newline at the end here.

Reply:
Sorry, missed this but I'll hit that soon